tasks/check-for-sessions.yml

```
@@ -0,0 +1,18 @@
+---
+  - name: Check for running meetings
+    uri:
+      url: "https://{{ bbb_hostname }}/bigbluebutton/api/getMeetings?checksum={{ ('getMeetings' + bbb_secret) | hash('sha1') }}"
+      return_content: true
+    register: bbb_getmeetings
+    ignore_errors: true
+    when: bbb_check_for_running_meetings
+    tags: always
+
+  - block:
+      - name: There are running sessions on this host
+        fail: msg="There are running meetings on {{ bbb_hostname }}, skipping."
+      - name: Stop play for Hosts with running meetings
+        meta: end_host
+    when: "bbb_check_for_running_meetings and not ansible_check_mode and not bbb_getmeetings.failed \
+           and '<messageKey>noMeetings</messageKey>' is not in bbb_getmeetings.content"
```

Proposal:
```suggestion
    when:
      - not ansible_check_mode
      - not bbb_getmeetings.failed
      - '<messageKey>noMeetings</messageKey>' is not in bbb_getmeetings.content"
```
https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html

```
You can use logical operators to combine conditions. When you have multiple conditions that all need to be true (that is, a logical and), you can specify them as a list:
~~~
tasks:
  - name: Shut down CentOS 6 systems
    ansible.builtin.command: /sbin/shutdown -t now
    when:
      - ansible_facts['distribution'] == "CentOS"
      - ansible_facts['distribution_major_version'] == "6"
~~~
```